docs/Code_commit_process/README.md

```diff
@@ -1,3 +1,35 @@
 # Test prerequisite for Code Commit
 
 ![Dent-SIT](../img/code_commit.png)
+
+To run the pre-commit checks locally, you can follow below steps:
+
+- Ensure that default python is python3.
+
+- Ensure that the `pre-commit` package is installed:
+
+```Shell
+sudo pip install pre-commit
+```
+
```

Comment:
There is also a [Debian package](https://packages.debian.org/pre-commit).

Reply:
i'll keep pypi package since those it is usually up to date.
one downside: too up to date may be to new and not work
if we see any problem we can change to debian package.